examples/multisession/BUILD

```diff
@@ -0,0 +1,31 @@
+load("//:cxx_abi_option.bzl", "CXX_ABI")
+
+load("@org_tensorflow//tensorflow:tensorflow.bzl", "tf_cc_binary")
+tf_cc_binary(
+    name = 'sample',
+    srcs = [
+        "sample.cpp",
+    ],
+    deps = [
+        "@org_tensorflow//tensorflow/cc:cc_ops",
+        "@org_tensorflow//tensorflow/cc:client_session",
+        "@org_tensorflow//tensorflow/core:tensorflow",
+        "//:ngraph_bridge_lib",
+        "@ngraph//:cpu_backend",
+    ],
+    copts = [
+        "-pthread", 
+        "-std=c++11", 
+        # "-D_FORTIFY_SOURCE=2",
+        # "-Wformat",
+        # "-Wformat-security",
+        # "-Wformat",
+        # "-fstack-protector-all",
+        "-D NDEBUG",
```

Comment:
This has `NDEBUG`. We should remove it right?

Reply:
Yes `NDEBUG` should be removed.
On another note Sample app is a duplicate of hello_tf, so it can be removed altogether.